CHANGELOG.rst

```diff
@@ -5,9 +5,12 @@
 
 *
 
-*
+* Add `strict` parameter to xfail marker. Thanks `@pquentin`_ for the
+  PR and `@nicoddemus`_ for noticing the issue.
```

Comment:
@The-Compiler I removed the entry from the changelog and mentioned "doc" in the commit.